core/src/test/java/org/apache/calcite/plan/volcano/VolcanoPlannerTest.java

```diff
@@ -254,6 +255,31 @@ public void onMatch(RelOptRuleCall call) {
     return sort(Arrays.asList(es));
   }
 
+  @Test void testSetMergeMatchSubsetRule() {
```

Comment:
Can you add a comment to explain what does this case do?

Reply:
sure, added.